### PR TITLE
Node 16

### PR DESCRIPTION
**Pull request by solomon-negusse**

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@ case "$1" in
     develop)
         echo "Running Development Server"
         echo -e "$GCLOUD_STORAGE" | base64 -d > storage.json
-        exec grunt --gruntfile app/Gruntfile.js | bunyan
+        exec yarn start | bunyan
```
Comment (tiagojsag) [inline entrypoint.sh:8]: this is unrelated... not sure what impact it may have

Comment (solomon-negusse) [reply entrypoint.sh:8]: This gets a node server running. The grunt command is throwing an error which will have to be tackled in a different PR. The downside is that this doesn't have live reload.. something that can be addressed with the proper tool when these MSs are updated.

**Review by tiagojsag — APPROVED** [review]
